contracts/MyMultiSig.cdc

```diff
@@ -252,6 +151,86 @@ pub contract MyMultiSigV2 {
             destroy action
         }
 
+        pub fun createMultiSign(action: {Action}): UInt64 {
+            let newAction <- create MultiSignAction(signers: self.signers.keys, intent: action.intent, action: action)
+            let uuid = newAction.uuid
+            self.actions[newAction.uuid] <-! newAction
+            emit ActionCreated(uuid: uuid, intent: action.intent)
+            return uuid
+        }
+
+        pub fun signerApproveAction(actionUUID: UInt64, messageSignaturePayload: MessageSignaturePayload): Bool {
+            pre {
+                self.signers[messageSignaturePayload.signingAddr] == true:
+                    "This address is not allowed to sign for this."
+                self.borrowAction(actionUUID:actionUUID).accountsVerified[messageSignaturePayload.signingAddr] != true:
+                    "This address has already signed."
+            }
+            let action = self.borrowAction(actionUUID:actionUUID)
+
+            // Validate Message
+            assert(
+                MyMultiSigV2.approveOrRevokeActionMessageIsValid(action: action, messageSignaturePayload: messageSignaturePayload),
+                message: "Signed message is invalid"
+            )
+        
+            // Validate Signature
+            var signatureValidationResponse = MyMultiSigV2.validateSignature(payload: messageSignaturePayload)
+            assert(signatureValidationResponse.isValid == true, message: "Invalid Signatures")
+            assert(signatureValidationResponse.totalWeight >= 999.0, message: "Total weight of combined signatures did not satisfy 999 key weight requirement.")
+
+            // Approve action
+            action.setAccountsVerified(signer: messageSignaturePayload.signingAddr, value: true)
+
+            emit ActionApprovedBySigner(address: messageSignaturePayload.signingAddr, uuid: self.uuid)
+            return signatureValidationResponse.isValid
```

Comment:
This will always be true, because of the assertion above, so I think it's safe to remove the return type for both signerApproveAction and signerRejectAction